<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>551</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Search by:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>95</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>551</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Userdata</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>95</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>200</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
